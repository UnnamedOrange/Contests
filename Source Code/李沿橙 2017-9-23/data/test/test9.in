5
22
1 1 2 3 4 5 6 7 8 9 10 11 11 13 14 15 19 42 1000 500 999 999 
50
1 2 3 4 5 6 7 8 9 10 101 102 103 104 105 106 107 108 109 110 201 202 203 204 205 206 207 208 209 210 301 302 303 304 305 306 307 308 309 310 501 502 503 504 505 506 507 508 509 510 
50
10 20 30 40 50 60 70 80 90 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 450 460 470 480 490 530 
42
1000 999 56 34 564 354 345 756 53 654 534 754 534 53 123 12 43 53 5 65 354 24 1 1 1 1 234 534 7 3 8 3 8 3 7 5 76 5 6 7 3 76 
50
437 417 122 474 451 353 209 347 832 51 95 680 746 515 102 612 918 768 209 926 56 469 784 159 663 25 785 802 740 161 45 997 263 285 508 667 94 167 893 467 540 183 278 76 57 876 237 903 347 814 
