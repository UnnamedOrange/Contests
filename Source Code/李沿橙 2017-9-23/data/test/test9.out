1 1 2 3 4 5 6 7 8 9 10 11 11 13 14 15 19 42 999 999 1000 500 
1 2 3 4 5 6 7 8 9 10 101 102 103 104 105 106 107 108 109 201 202 203 204 205 206 207 208 209 210 301 303 304 305 306 307 308 309 501 503 504 505 506 507 508 509 510 502 310 302 110 
10 20 30 40 50 70 80 90 100 120 130 140 150 170 180 190 200 220 230 240 250 270 280 290 300 320 330 340 350 370 380 390 400 420 430 440 450 470 480 530 490 460 410 360 310 260 210 160 110 60 
1 1 1 1 3 3 3 3 5 5 5 6 7 7 7 8 8 12 24 34 43 53 53 53 56 65 76 76 123 345 354 534 534 534 654 754 999 1000 756 564 354 234 
25 45 51 56 57 76 94 95 102 159 161 167 183 209 209 263 278 285 347 347 417 437 451 467 469 508 515 612 667 680 740 746 768 784 802 814 832 876 893 918 926 997 903 785 663 540 474 353 237 122 
